### PR TITLE
Write CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by Monstarrrr**

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+Examples of unacceptable behavior include:
+
+* The use of sexualized language or imagery, and sexual attention or
+  advances of any kind
+* Trolling, insulting or derogatory comments, and personal or political attacks
+* Public or private harassment
+* Publishing others' private information, such as a physical or email
+  address, without their explicit permission
+* Other conduct which could reasonably be considered inappropriate in a
+  professional setting
```
Comment (purple-void) [inline CODE_OF_CONDUCT.md:37]: "professional setting" is vague and confusing. maybe rephrase this?

File: CODE_OF_CONDUCT.md
Comment (Monstarrrr) [inline CODE_OF_CONDUCT.md:1]: The file is at the wrong location, needs to be in ``/docs``

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (Monstarrrr) [reply CODE_OF_CONDUCT.md]: Removed & edited

**Review by purple-void — APPROVED** [review]
